packages/DropdownMenu/stories/DropdownMenu.stories.styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { Story } from "storybook/assets/styles/common.styles";
+
+// Common DropdownMenu story styles
+
+export const DropdownMenuStory = styled(Story)`
+  max-width: 500px;
```

Comment:
Do you need to confine the story to 500px? If not, you can skip this whole file and just use `Story` directly in your Showcase story.

Reply:
I've left this in as it gives us some padding to the edge of the screen, otherwise the popup content will display slightly offset from the side of the screen which looks odd.

Reply:
Oh, I see! A hard-coded width will only sometimes work though, depending on the size of the window / width of add-on panel... `padding-right` is probably a better approach, no?

Reply:
@tristanjasper did you address this?